compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -545,13 +545,36 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
         // `[[fn#'b -> impl_trait#'b], [fn#'b -> impl_sized#'b]]`
```

Comment:
Why isn't the mapping `[[fn#'b -> impl_trait#'b], [impl_trait#'b -> impl_sized#'b]]`?

Reply:
After my new change would always be `[[fn#'b -> impl_trait#'b], [fn#'b -> impl_sized#'b]]`